ast/Trees.h

```diff
@@ -675,6 +675,11 @@ class EmptyTree final : public Expression {
  * Desugar string concatenation into series of .to_s calls and string concatenations
  */
 
+// TODO Find a better place for this
```

Comment:
Wasn't sure where this should live

Reply:
`ast/Helpers.h` is the best I can think of in 30 secs